floor_generator/lib/value_object/dao.dart

```diff
@@ -1,11 +1,12 @@
 import 'package:analyzer/dart/element/element.dart';
 import 'package:floor_generator/value_object/deletion_method.dart';
-import 'package:floor_generator/value_object/entity.dart';
 import 'package:floor_generator/value_object/insertion_method.dart';
 import 'package:floor_generator/value_object/query_method.dart';
 import 'package:floor_generator/value_object/transaction_method.dart';
 import 'package:floor_generator/value_object/update_method.dart';
 
+import 'entity.dart';
```

Comment:
💡We could get rid of all the unnecessary import qualifiers.

```dart
import 'deletion_method.dart';
import 'entity.dart';
import 'insertion_method.dart';
import 'query_method.dart';
import 'transaction_method.dart';
import 'update_method.dart';
```

Reply:
Then we should do it everywhere... but I think this should be part of another PR... I'll just put the entity import back where it was for now to make int consistent with the other statements...